### PR TITLE
11361 - API Management Portal

### PR DESCRIPTION
**Pull request by MarcGueury**
# 11361 - API Management Portal
Also small changes in 11272 - Search Documents using OpenSearch, AI Vision, Text Recognition

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

### CODE REVIEW

File: oci-api-management/existing-apis/existing-apis.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
+
+# Lab 2: Add Existing APIs
+
+## Introduction
+
```
Comment (anooshapilli) [inline oci-api-management/existing-apis/existing-apis.md:5]: @MarcGueury Please add a description that covers an overview of the lab in the introduction section for all the labs.

File: oci-api-management/existing-apis/existing-apis.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+- Add existing API from Oracle Integration:
+    - Create Samples APIs in OIC 
+    - Configure the security
+    - Collect the APIs
+- Add existing API from Oracle API Gateway
+
+### Prerequisites
+
+Follow previous labs.
```
Comment (anooshapilli) [inline oci-api-management/existing-apis/existing-apis.md:18]: @MarcGueury - Make this a bullet point

File: oci-api-management/existing-apis/existing-apis.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+## Task 1: User and Tenancy details
+
+First, we need data about your tenancy and user.
```
Comment (anooshapilli) [inline oci-api-management/existing-apis/existing-apis.md:22]: @MarcGueury - Number all the instructions in a task and indent the bullets, code blocks, and images with 4 spaces from the beginning of the line for all the tasks in all the labs.

File: oci-api-management/intro/intro.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+
+# Introduction
+
+Estimated time: 60 minutes
+
+### Objective
```
Comment (anooshapilli) [inline oci-api-management/intro/intro.md:6]: @MarcGueury There is another objective section at the end of this lab. Please consolidate them into one.

File: oci-api-management/intro/intro.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+Create an API Management Portal 
+
+### Features
```
Comment (anooshapilli) [inline oci-api-management/intro/intro.md:10]: @MarcGueury - You can make this heading 2 ## and rename it to About this Workshop

File: oci-api-management/intro/intro.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
+
+# Introduction
+
+Estimated time: 60 minutes
```
Comment (anooshapilli) [inline oci-api-management/intro/intro.md:4]: @MarcGueury - Move this line before the ### Architecture section

**Review by anooshapilli — COMMENTED** [review]
@MarcGueury - There are changes made to the opensearch-objectstorage folder that is not part of this workshop. Could you update the WMS ID for that workshop?

Please make the requested changes to the workshop.

File: oci-api-management/existing-apis/existing-apis.md (hunk 1 above)
Comment (MarcGueury) [reply oci-api-management/existing-apis/existing-apis.md:5]: done

File: oci-api-management/existing-apis/existing-apis.md (hunk 2 above)
Comment (MarcGueury) [reply oci-api-management/existing-apis/existing-apis.md:18]: Done

File: oci-api-management/existing-apis/existing-apis.md (hunk 3 above)
Comment (MarcGueury) [reply oci-api-management/existing-apis/existing-apis.md:22]: Done

File: oci-api-management/intro/intro.md (hunk 4 above)
Comment (MarcGueury) [reply oci-api-management/intro/intro.md:6]: Renamed the last sections "steps "

File: oci-api-management/intro/intro.md (hunk 5 above)
Comment (MarcGueury) [reply oci-api-management/intro/intro.md:10]: Done

File: oci-api-management/intro/intro.md (hunk 6 above)
Comment (MarcGueury) [reply oci-api-management/intro/intro.md:4]: ? Not sure, it is before the ### Architecture section already. Skipping it.

**Comment by MarcGueury** [comment]
> @MarcGueury - There are changes made to the opensearch-objectstorage folder that is not part of this workshop. Could you update the WMS ID for that workshop?

I have no way to split the change. I would need to rollback the change of my other lab... :-(
Anyway, the OpenSearch lab id is WMS 11272

**Comment by MarcGueury** [comment]
Please review the changes. Thanks

**Review by MarcGueury — COMMENTED** [review]
Done. Changes are committed in my branch. 
 am not sure if I need to recreate a new Pull request ?

**Comment by anooshapilli** [comment]
Once you commit the changes to your repo, they would be added to this PR.

**Review by bhenndricks — APPROVED** [review]

**Comment by anooshapilli** [comment]
@MarcGueury - Just following up on the requested changes.

**Comment by MarcGueury** [comment]
Not sure what is missing ? I did all the change requested ? or not ?

**Review by malcherm — APPROVED** [review]

**Pull request merged**